README.md

```diff
@@ -72,14 +72,33 @@ check.addCheck('cassandra', 'timeout', async () => {
         metricValue: 250,
         metricUnit: "ms"
     };
-});
+}, 10000);
 
 // Add middleware to your Express app:
 app.use(check.express());
 app.listen(3535);
 ```
 
-### Example for Koa.js
+### Caching and Backend Protection
+
+Please note the third argument (`10000`) in the `.addCheck()` call for the
```

Comment:
Well, fourth, but the idea is the important thing. /tomlehrer

Reply:
Good catch. Fixed by 7d24e7e